server/store/datastore/migration/000_legacy_to_xorm.go

```diff
@@ -96,7 +96,7 @@ var legacy2Xorm = task{
 				return err
 			}
 			if _, err := sess.Exec("INSERT INTO build_config (config_id, build_id) SELECT config_id,build_id FROM old_build_config;"); err != nil {
-				return fmt.Errorf("unable to set copy data in to temp table %s. Error: %v", "old_build_config", err)
+				return fmt.Errorf("unable to set copy data into temp table %s. Error: %v", "old_build_config", err)
```

Comment:
This is my newest correction and has been the most controversial. Happy to drop. That said, given that the code itself uses `INTO`, I'm relatively confident in it.